platform/opensuse/kanidm-unixd-tasks.service

```diff
@@ -3,8 +3,10 @@
 
 [Unit]
 Description=Kanidm Local Tasks
-After=chronyd.service ntpd.service network-online.target suspend.target kanidm-unixd.service
-Wants=kanidm-unixd.service
+After=chronyd.service ntpd.service network-online.target kanidm-unixd.service
+# This prevents starting unixd-tasks before unixd is running and
+# has created the socket necessary for communication.
+ConditionPathExists=/var/run/kanidm-unixd/task_sock
```

Comment:
/var/run is more legacy and on modern systems just a symlink to /run, I suggest to reference /run directly instead.

Reply:
True, need to check if OpenSUSE leap has made that change yet. They for example haven't merged /bin & /usr/bin yet.

Reply:
I am coming from (open)SUSE. :)